Ports/SonicMania/package.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port='SonicMania'
+version='1.05.0713'
+workdir="RSDKv5-Decompilation-1b01682d22948aaa3971e96f20f038c144b7eaf4"
+launcher_name='Sonic Mania'
+launcher_category='&Games'
+launcher_command='/usr/local/share/games/sonicmania/RSDKv5U'
+icon_file='RSDKv5U/RSDKv5U.ico'
+depends=(
+  'SDL2'
+  'SDL2_sound'
+  'SDL2_mixer'
+  'libtheora'
+  'libogg'
+  'stb'
+)
+files=(
+    "https://github.com/Rubberduckycooly/RSDKv5-Decompilation/archive/1b01682d22948aaa3971e96f20f038c144b7eaf4.tar.gz#b12ec54e5d763a4a48b1f4a3ca19cca4441c6efdaa95d77614dc82d1ab11b916"
```

Comment:
why are we downloading an  arbitrary commit of this package instead of a tagged version?

Reply:
right we are downloading a specific commit i'll look into it later